src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/services/RadicleProjectService.java

```diff
@@ -86,16 +87,26 @@ public ProcessOutput getVersion(String path) {
         }
     }
 
-    public ProcessOutput track(GitRepository root, String reference, String nodeUrl) {
+    public ProcessOutput trackRepo(RadTrack.Repo repo) {
         List<String> args = new ArrayList<>();
         args.add("track");
-        if (!Strings.isNullOrEmpty(reference)) {
-            args.add(reference);
+        args.add(repo.rid());
+        if (!repo.scope().equals(RadTrack.Scope.NONE)) {
+            args.add("--scope");
+            args.add(repo.scope().name.toLowerCase());
```

Comment:
This is invalid. 
The name of the scope is received from the bundle, which might be completely different from what you need, e.g. in case of i18n and multiple languages.